client/package.json

```diff
@@ -24,44 +24,32 @@
     "redux-logger": "^3.0.6",
     "redux-thunk": "^2.3.0",
     "smoothscroll-polyfill": "^0.4.4",
+    "use-local-storage": "^2.1.1",
     "uuid": "^8.3.2",
     "yn": "^4.0.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.13.15",
-    "@hot-loader/react-dom": "^17.0.1",
+    "@types/node": "^14.14.41",
     "@types/react": "^17.0.3",
     "@types/react-dom": "^17.0.3",
     "@types/react-redux": "^7.1.16",
     "@types/react-scroll": "^1.8.2",
     "@types/smoothscroll-polyfill": "^0.3.1",
+    "@types/react-beautiful-dnd": "^13.0.0",
     "@types/uuid": "^8.3.0",
     "@typescript-eslint/eslint-plugin": "^4.22.0",
     "@typescript-eslint/parser": "^4.22.0",
-    "babel-eslint": "10.x",
-    "babel-preset-gatsby": "^1.2.0",
-    "eslint": "7.x",
-    "eslint-config-prettier": "^8.1.0",
-    "eslint-config-react-app": "^6.0.0",
-    "eslint-loader": "^3.0.2",
-    "eslint-plugin-flowtype": "5.x",
-    "eslint-plugin-import": "^2.22.1",
-    "eslint-plugin-jsx-a11y": "^6.4.1",
-    "eslint-plugin-prettier": "^3.3.1",
+    "eslint": "^7.25.0",
+    "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-react": "^7.23.2",
-    "eslint-plugin-react-hooks": "4.x",
-    "gatsby-cli": "^3.2.0",
     "gatsby-plugin-env-variables": "^2.1.0",
     "gatsby-plugin-eslint": "^3.0.0",
-    "gatsby-plugin-offline": "^4.2.0",
-    "gatsby-plugin-root-import": "^2.0.6",
-    "gatsby-plugin-sharp": "^3.2.1",
-    "gatsby-plugin-typescript": "^3.2.0",
-    "identity-obj-proxy": "^3.0.0",
+    "gatsby-plugin-manifest": "^3.3.0",
```

Comment:
let's get rid of plugin manifest altogether. Among other things it forces us to have a favicon and we don't have a working solution for making that configurable